R/AzureDataLake.R

```diff
@@ -0,0 +1,368 @@
+
+#' Azure Data Lake LISTSTATUS for specified relativePath of an azure data lake account.
+#'
+#' @inheritParams createAzureContext
+#' @inheritParams azureAuthenticate
+#'
+#' @param azureActiveContext Provide an `azureActiveContext` object used for authentication.
+#' @param azureDataLakeAccount Provide the name of the Azure Data Lake account.
+#' @param relativePath Provide a relative path of the directory.
+#' @param verbose Print tracing information (default FALSE).
+#'
+#' @return Returns a data frame.
```

Comment:
But what's _in_ the data frame? One row per directory? What are the columns? And error conditions?

Reply:
Fixed in https://github.com/omkarksa/AzureSMR/pull/1 will raise a pull request once this is merged into master